src/unfold/compat.py

```diff
@@ -0,0 +1,15 @@
+import django
+from django.contrib.admin.views.main import ERROR_FLAG, PAGE_VAR
+from django.contrib.admin.views.main import ChangeList as DjangoChangeList
+
+if django.VERSION < (5, 0):
```

Comment:
Once we are not checking for Django version, maybe we can move this new class to the views.py exactly as the default Changelist:

https://github.com/django/django/blob/5a1cae3a5675c5733daf5949759476d65aa0e636/django/contrib/admin/views/main.py#L65